packages/effects-core/src/math/value-getters/bezier-curve.ts

```
@@ -0,0 +1,379 @@
+}
+
+export class BezierCurve extends ValueGetter<number> {
+  curveMap: Record<string, CurveInfo>;
+  keys: number[][];
+
+  private keyFrames: Keyframe[];
+  private curveInfos: CurveInfo[];
+
+  constructor (arg: any) {
+    super(arg);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)